makeflow/src/makeflow.c

```diff
@@ -149,6 +149,8 @@ static char *mountfile = NULL;
 static char *mount_cache = NULL;
 static int use_mountfile = 0;
 
+static char *umbrella_log_prefix = NULL;
```

Comment:
This prefix belongs in the umbrella wrapper, not in makeflow.c. Refer to how we handle logs for the resource monitor.